### ISSUE TITLE
Missing Method Exception being thrown during Revit Lookup "OnStartup" execution

### ISSUE THREAD

**Issue by chrishildebran** [jeremytammik/RevitLookup#262]
Weve been getting this error after upgrading to the latest version of Revit Lookup.

It is happening for Revit versions:

- 2021
- 2022
- 2023
- 2024

![image](https://github.com/jeremytammik/RevitLookup/assets/61524205/49b461cb-0d46-4537-834b-57c525f9a390)

**Comment by Nice3point** [comment]
Versions older than 2025 may conflict with installed plugins. Serilog used by your other plugins should be updated, RevitLookup uses the latest. This will help to resolve this issue. In the other case, use 2025, which implements dependency isolation

**Comment by chrishildebran** [comment]
This seems to be quite a blunt answer Roman. 

I wish it was a simple as upgrading all of our projects to Revit 2025. We are a large company with dozens of projects in all of the Revit versions listed above specified in large part by our customers. Weve come to rely on Revit Lookup since Revit 2017 and have never had a problem until this one. 

Is there no other way to remedy this without causing such a hassle for other developers?

**Comment by chrishildebran** [comment]
Roman, I appreciate your contributions to the Revit community and Jeremy, but that answer isn't acceptable or useful. 

As I'm sure many others do, my company uses all Revit versions back to 2021 because we have multiple customers with varying project specifications. Software development cycles also play a major part in this situation. 

How can i possibly control what other packages other addin developers use?

Can you rephrase your answer in a more friendly and helpful way that addresses more than just Revit 2025?

**Comment by Nice3point** [comment]
RevitLookup uses third-party dependencies designed by other developers. Other plugins may also use these dependencies, but of different versions, which will cause a conflict and cause an error when running the plugin. I use the most recent and up-to-date versions for RevitLookup, the plugins you use may contain outdated versions. We have no control over this process, and we cannot prevent other developers from using third-party libraries. Why are you getting this error in RevitLookup and not in another plugin? Because plugins are loaded in alphabetical order, and the 'R' is near the end, so that's where you get the exception and RevitLookup not working. Unfortunately, whose plugin is loaded first decides which version of the dependency to use. And if it's outdated, RevitLookup, which uses the newest one, will simply be incompatible, if everything was in reverse order this could have been avoided. At the moment there are two ways, disable conflicting plugins, or use Revit 2025 where this problem does not exist, and plugins can isolate their dependencies from each other. For Revit 2024 and older we can't influence it, not in any way
